File: Praktikum2/Java/Product.java
class Product {
    private int price;
    private int idProduct;

    Product() {
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public int getPrice() {
        return this.price;
    }

    public void setIdProduct(int idProduct) {
        this.idProduct = idProduct;
    }

    public int getIdProduct() {
        return this.idProduct;
    }

    public void printProduct() {
        System.out.println("Id Product : " + getIdProduct());
        System.out.println("Price : " + getPrice());
    }
}

File: Praktikum2/C++/Product.cpp
#include <bits/stdc++.h>

using namespace std;

class Product
{
private:
    int price;
    int idProduct;

public:
    Product() {}

    void setPrice(int price)
    {
        this->price = price;
    }

    int getPrice()
    {
        return this->price;
    }

    void setIdProduct(int idProduct)
    {
        this->idProduct = idProduct;
    }

    int getIdProduct()
    {
        return this->idProduct;
    }

    void printProduct()
    {
        cout << "Id Product : " << getIdProduct() << endl;
        cout << "Price : " << getPrice() << endl;
    }

    ~Product() {}
};